docs/tutorials/dc/create-tag-property.md

```diff
@@ -123,4 +123,4 @@ In this section we will create a mobile property which contains the configuratio
 | :---: |
 | **View install instructions** |
 
-For the next part of the tutorial, we will show how to author an in-app message in **Adobe Journey Optimizer**.
\ No newline at end of file
+For the next part of the tutorial, we will show how to create the channel configuration in **Adobe Journey Optimizer**.
```

Comment:
do we also need to update this part in the datastream doc, to point to creating a tag property next (which is a new order for this update)?

Reply:
The datastream page didn't have a pointer to the surface page. I added one to point to tag property step